.ocamlformat

```diff
@@ -0,0 +1 @@
+version=0.15.0
```

Comment:
That's a bit old on ocamlformat world I understand, should we use a more recent version?

Additionally, I'd specify `profile=ocamlformat` explicitly [or the desired alternative] For reference this is a style I tend to enjoy:
```
module-item-spacing=compact
sequence-style=terminator
cases-exp-indent=2
field-space=loose
exp-grouping=preserve
break-cases=fit
doc-comments=before
```
but of course YMMV.

Reply:
> That's a bit old on ocamlformat world I understand, should we use a more recent version?

That's what I tried in the first place but it turns out there are some conflicts

<img width="661" alt="Screenshot 2021-05-17 at 21 38 01" src="https://user-images.githubusercontent.com/5595092/118548603-c7aee680-b75a-11eb-8eec-5cdc36d50925.png">

I guess once #54 get merged, we should be able to use the latest version :)

> Additionally, I'd specify `profile=ocamlformat` explicitly [or the desired alternative] For reference this is a style I tend to enjoy:
> 
> ```
> module-item-spacing=compact
> sequence-style=terminator
> cases-exp-indent=2
> field-space=loose
> exp-grouping=preserve
> break-cases=fit
> doc-comments=before
> ```
> 
> but of course YMMV.

I'll make the changes 👍

Reply:
Note that I posted this style just a guideline, if you or @gasche do prefer a different one I'm 100% fine with it.

I will leave a bit more of time before merging in case people would like to comment, then merge [and if you would like to squash the commits that'd be fine too]

Reply:
I don't have any personal preferences. As long as I don't have to manually format that's good for me 🤗 Thanks for the review, I'll squash the commits when I get back to my computer 👍